### ISSUE TITLE
Redirects with query params

### ISSUE THREAD

**Issue by zigomir** [netlify/netlify-dev-plugin#147]
```
netlify -v
netlify-cli/2.11.11 darwin-x64 node-v10.15.3
```

Seems like query params in redirects are not working with `netlify dev`.
I tried same config on prod with `netlify deploy` and it works fine there.

Contents of my `_redirects`

```
/oauth_callback state=:state code=:code /.netlify/functions/login?code=:code&state=:state 301
```

**Comment by sw-yx** [comment]
thanks for the bug report!

**Comment by kangw3n** [comment]
experience same problem here 👍

**Comment by sw-yx** [comment]
we aren't actively working on this issue right now, if anyone can figure it out PRs are welcome

**Comment by zigomir** [comment]
But this will get resolved before you you go out of beta?

**Comment by sw-yx** [comment]
i dont control the roadmap. we're in the process of hiring to build out Netlify Dev. if you know anyone keen to work on this, refer them!

**Comment by sw-yx** [comment]
related internal convo: https://netlify.slack.com/archives/CGW2Y6XPH/p1557955825318700

related internal issue: https://github.com/netlify/bitballoon/issues/212#issuecomment-496441898

**Comment by shortdiv** [comment]
Also this was logged as an issue in the private [rules-proxy lib](https://github.com/netlify/netlify-rules-proxy/issues/4) and might be an issue with the internal redirector lib that parses and creates redirect rules.

**Comment by zigomir** [comment]
hmm. both links are 404 for me ;)

**Comment by shortdiv** [comment]
Yeah, it's a private repo. Mentioned it so it'd be useful for internal Netlify folks to know where to look when addressing this later on
